Core/Validators/ProductDTOValidator.cs

```diff
@@ -0,0 +1,16 @@
+﻿using FluentValidation;
+using LearnNet_CatalogService.Core.DTO;
+
+namespace LearnNet_CatalogService.Core.Validators
+{
+    public class ProductDTOValidator : AbstractValidator<ProductDTO>
+    {
+        public ProductDTOValidator()
+        {
+            RuleFor(x => x.Name).NotEmpty().MaximumLength(50);
+            RuleFor(x => x.CategoryId).NotEmpty();
+            RuleFor(x => x.Price).NotEmpty().GreaterThanOrEqualTo(0);
+            RuleFor(x => x.Amount).NotEmpty().GreaterThanOrEqualTo(0);
```

Comment:
Is there a scenario where a user can order 0 items?

Reply:
maybe no, agree